src/frontend/src/pages/CalendarPage/DesignReviewDetailPage/FinalizeDesignReviewModal.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  Box,
+  FormControl,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  ToggleButton,
+  ToggleButtonGroup,
+  Typography
+} from '@mui/material';
+import NERModal from '../../../components/NERModal';
+import { useState } from 'react';
+
+interface FinalizeDesignReviewProps {
+  open: boolean;
+  setOpen: (val: boolean) => void;
+  onSubmit?: () => void;
+}
+
+const FinalizeDesignReviewModal = ({ open, setOpen, onSubmit }: FinalizeDesignReviewProps) => {
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const names = ['jdel', 'dehkdh', 'dehjkdhk', 'dsahksdh', 'dsajkldsaj'];
```

Comment:
this should be a prop that gets passed in. also that section of the modal should only show up if there are any conflicting meetings to show